policyengine_canada/variables/gov/cra/benefits/dental_benefit/dental_benefit.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class dental_benefit(Variable):
+    value_type = float
+    entity = Person
+    label = "Canada dental benefit"
+    unit = CAD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        income = person.household("adjusted_family_net_income", period)
+        p = parameters(period).gov.cra.benefits.dental_benefit
+        eligible = person("dental_benefit_eligible", period)
+        full_custody_amount = p.amount.calc(income)
+        # Divide by two if shared custody.
+        full_custody = person("full_custody", period)
+        amount = full_custody_amount / where(full_custody, 1, 2)
```

Comment:
Let's actually make the 50% a policy parameter, as it's specified in subsection (9)(2) of the Dental Benefit Act: https://laws-lois.justice.gc.ca/eng/acts/D-1.55/page-1.html